.evergreen/mongodl.py

```diff
@@ -800,9 +800,9 @@ def _latest_build_url(
     edition: str,
     component: str,
     branch: "str|None",
-) -> str:
+) -> tuple[str, str]:
     """
-    Get the URL for an "unpublished" "latest" build.
+    Get the URL and SHASUM for an "unpublished" "latest" build.
```

Comment:
```suggestion
    Get the URL for an "unpublished" "latest" build.
```
No checksum for latest builds.